pxtblocks/blocklyloader.ts

```diff
@@ -2378,6 +2378,46 @@ namespace pxt.blocks {
 
         msg.FUNCTIONS_CALL_TITLE = functionCall.block["FUNCTIONS_CALL_TITLE"];
         installBuiltinHelpInfo(functionCallId);
+        installBuiltinHelpInfo("function_call_output");
```

Comment:
should probably be a const for `"function_call_output"` defined wherever `functionCallId` is?

Reply:
It's only used here, so I skipped the variable declaration